Introducing the jira software plugin for jenkins

rafalmyslek opened the pull request:
Add blog post: "Introducing the Jira Software plugin for Jenkins"

MarkEWaite commented on content/blog/2019/08/2019-08-30-introducing-the-jira-software-plugin-for-jenkins.adoc:
```diff
@@ -0,0 +1,201 @@
+. Login to your Jenkins server and navigate to the Plugin Manager.
+
+. Select the 'Available' tab and search for 'Atlassian Jira Software Cloud' as the plugin name then install it.
+* The open-source plugin is hosted at on the Jenkins GitHub account. link:https://github.com/jenkinsci/atlassian-jira-software-cloud-plugin[You can check it out here].
```
Extra words possibly?
```suggestion
* The open-source plugin is hosted in the Jenkins GitHub account. link:https://github.com/jenkinsci/atlassian-jira-software-cloud-plugin[You can check it out here].
```

MarkEWaite commented on content/blog/2019/08/2019-08-30-introducing-the-jira-software-plugin-for-jenkins.adoc:
```diff
@@ -0,0 +1,201 @@
+
+*For sending deployment information*
+
+This is an example snippet of two sages that runs on any change to the staging or master branch. Again, we use a post step to send deployment data to Jira and the relevant issues. Here, the *environmentId*, *environmentName*, and *environmentType* need to be set to whatever you want to appear in Jira.
```
Minor spelling fix.
```suggestion
This is an example snippet of two stages that run on any change to the staging or master branch. Again, we use a post step to send deployment data to Jira and the relevant issues. Here, the *environmentId*, *environmentName*, and *environmentType* need to be set to whatever you want to appear in Jira.
```

MarkEWaite commented on content/blog/2019/08/2019-08-30-introducing-the-jira-software-plugin-for-jenkins.adoc:
```diff
@@ -0,0 +1,201 @@
+
+== Questions or feedback?
+
+If you have any questions, please link:https://support.atlassian.com/contact/[Atlassian support] and they will route it to the correct team to help you.
```
Missing a word in the sentence?
```suggestion
If you have any questions, please contact link:https://support.atlassian.com/contact/[Atlassian support] and they will route it to the correct team to help you.
```

oleg-nenashev commented on content/_data/authors/rafalmyslek.adoc:
```diff
@@ -0,0 +1,4 @@
+---
+name: "Rafal Myslek"
+github: rafalmyslek
+---
```
Add a short bio?

oleg-nenashev commented on content/blog/2019/08/2019-08-30-introducing-the-jira-software-plugin-for-jenkins.adoc:
```diff
@@ -0,0 +1,201 @@
+author: rafalmyslek
+---
+
+NOTE: This is a guest post by Rafal Myslek from link:https://www.atlassian.com/jira[Atlassian].
```
Not sure this note is needed in this case

oleg-nenashev commented on content/blog/2019/08/2019-08-30-introducing-the-jira-software-plugin-for-jenkins.adoc:
```diff
@@ -0,0 +1,201 @@
+
+According to a link:https://www.atlassian.com/blog/software-teams/modern-software-development-trends[recent survey] we conducted, software & IT teams on average use 4+ tools to move code from development to customer-facing production. As a result, teams struggle with keeping the status of work updated and understanding the overall health of their delivery pipeline.
+
+To solve this problem, I am excited to announce that we built link:https://plugins.jenkins.io/atlassian-jira-software-cloud[an official plugin between Jenkins and Jira Software Cloud]. The plugin automatically associates build and deployment information from Jenkins with relevant Jira issues and exposes key information about your pipeline across Jira issues, boards and via JQL. This means you can ...
```
```suggestion
To solve this problem, I am excited to announce that we built link:https://plugins.jenkins.io/atlassian-jira-software-cloud[an official Jenkins plugin for Jira Software Cloud]. The plugin automatically associates build and deployment information from Jenkins with relevant Jira issues and exposes key information about your pipeline across Jira issues, boards and via JQL. This means you can use Jira Software to automatically update and track issues through your complete development pipeline, from backlog to release.
```

rafalmyslek commented:
Great thanks for the review @MarkEWaite and @oleg-nenashev . I've applied the suggested changes, resized the image, squashed the commits. Also, I've moved the blog post to September.

oleg-nenashev commented on content/blog/2019/09/2019-09-02-introducing-the-jira-software-plugin-for-jenkins.adoc:
```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "Introducing the Jira Software plugin for Jenkins"
```
```suggestion
title: "Introducing the Jira Software Cloud plugin for Jenkins"
```

oleg-nenashev commented:
Will merge tomorrow if no negative feedback

daniel-beck commented:
⚠️  needs date update now.
